packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -284,14 +284,21 @@ export class OdspDelayLoadedDeltaStream {
 			if (hasFacetCodes(e) && e.facetCodes !== undefined) {
 				for (const code of e.facetCodes) {
 					switch (code) {
+						case "sessionForbidden":
 						case "sessionForbiddenOnPreservedFiles":
 						case "sessionForbiddenOnModerationEnabledLibrary":
 						case "sessionForbiddenOnRequireCheckout":
+						case "sessionForbiddenOnCheckoutFile":
+						case "sessionForbiddenOnInvisibleMinorVersion":
 							// This document can only be opened in storage-only mode.
 							// DeltaManager will recognize this error
 							// and load without a delta stream connection.
 							this.policies = { ...this.policies, storageOnly: true };
-							throw new DeltaStreamConnectionForbiddenError(code, { driverVersion });
+							throw new DeltaStreamConnectionForbiddenError(
```

Comment:
seems weird to pass the same value twice, why not just reuse the existing parameter.

Reply:
Changed the message. Added the argument as I don't want to use the message to compare etc as generally error message is something which is changeable, unlike code.